<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="3">
     <widget class="QPushButton" name="portMonitorPushButton">
      <property name="text">
       <string>open port monitor</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="setParamsPushButton">
      <property name="text">
       <string>Set parametrs</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="showStatusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>0 - 4095</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="gainLabel">
      <property name="text">
       <string>Gain</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="triggerLabel">
      <property name="text">
       <string>Trigger level</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="GainLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="triggerLowLineEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="portNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="portNameLabel">
      <property name="text">
       <string>port name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="connectPushButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
